<launch>
    <node pkg="maze_control" type="spheromini_control.py" name="spheromini_control" output="screen">
    <rosparam command="load" file="$(find maze_control)/param/sphero_control.yaml"/>
    </node>
</launch>
